src/regtech_data_validator/phase_validations.py

```diff
@@ -182,11 +184,11 @@ def get_phase_1_and_2_validations_for_lei(context: dict[str, str] | None = None)
                     has_no_conditional_field_conflict,
                     id="E2000",
                     name="ct_credit_product_ff.conditional_field_conflict",
-                    description=(
-                        "When 'credit product' does not equal 977 (other), 'free-form"
-                        " text field for other credit products' must be blank."
-                        " When 'credit product' equals 977, 'free-form text field "
-                        "for other credit products' must not be blank."
+                    description=dedent(
+                        """\
+                        * When 'credit product' does **not** equal 977 (other), 'free-form text field for other credit products' must be blank.
+                        * When 'credit product' equals 977, 'free-form text field for other credit products' must **not** be blank.
+                    """
```

Comment:
Did `black` format it this way, with the offset `"""`s?

Reply:
Yup